aiida/tools/importexport/migration/v03_to_v04.py

```diff
@@ -457,7 +457,7 @@ def migrate_v3_to_v4(metadata, data, folder, *args):  # pylint: disable=unused-a
     migration_dbgroup_type_string_change_content(data)
     migration_calc_job_option_attribute_keys(data)
     migration_move_data_within_node_module(data)
-    migration_trajectory_symbols_to_attribute(data, folder)
+    migration_trajectory_symbols_to_attribute(data, args[0])
```

Comment:
Wouldn't it be more clear to then define `folder = args[0]` above and pass it here? I know it isn't stringently necessary, it just seems like a small effort with large gain for future code readers?